$OpenBSD: patch-panels_info-overview_meson_build,v 1.4 2021/10/30 09:14:55 ajacoutot Exp $

Fix build with meson >= 0.60.0

No udisks2 on OpenBSD (XXX add HAVE_UDISKS2 to meson_options.txt)

Index: panels/info-overview/meson.build
--- panels/info-overview/meson.build.orig
+++ panels/info-overview/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -40,7 +39,6 @@ sources += gnome.compile_resources(
 )
 
 deps = common_deps + [
-  dependency('udisks2', version: '>= 2.8.2'),
   dependency('libgtop-2.0')
 ]
 
